x/protorev/keeper/routes_test.go

```diff
@@ -129,6 +130,8 @@ func (s *KeeperTestSuite) TestBuildRoutes() {
 
 // TestBuildHighestLiquidityRoute tests the BuildHighestLiquidityRoute function
 func (s *KeeperTestSuite) TestBuildHighestLiquidityRoute() {
+	// TODO:
```

Comment:
Accidental TODO?

Reply:
Ah yes, I fixed what I made that todo for, ty